extensions/arc/src/ui/dashboards/postgres/postgresParameters.ts

```diff
@@ -607,14 +612,31 @@ export abstract class PostgresParametersPage extends DashboardPage {
 		});
 	}
 
+	private refreshParametersTable(): void {
```

Comment:
I'd add a comment somewhere that this only updates existing values, and won't add new parameters to the table if they don't already exist.